lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Organization
+field_name_base_url: http://hl7.org/fhir/r4/organization-definitions.html#Organization
```

Comment:
All the links in this file, should be secured.

Reply:
done https://github.com/cerner/fhir.cerner.com/pull/325/commits/981f8f421bb02f636415ef251b731508616a52d6